modules/persistent-storage-csi-vsphere-storage-class.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vpshere.adoc
+
+[id="csi-vsphere-storage-class"]
+= Creating the vSphere CSI storage class
+
+Storage classes are used to differentiate and delineate storage levels and
+usages. By defining a storage class, users can obtain dynamically provisioned
+persistent volumes.
+
+.Procedure
+
+. In the {product-title} console, click *Storage* -> *Storage Classes*.
+
+. In the storage class overview, click *Create Storage Class*.
+
+. In the *StorageClass* page, click *Edit YAML*.
+
+. Edit the YAML file as shown below:
++
+[source,yaml]
+----
+kind: StorageClass
+  apiVersion: storage.k8s.io/v1
+  metadata:
+    name: <storage-class> <1>
+    annotations:
+      storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
We should not recommend customers to create additional default classes like this (user must disable old default SC first) . This could result is various problems. Lets drop this annotation.

Reply:
This file is not part of the content anymore, but I didn't delete it. I've deleted it now, so it shouldn't show up in GH.